File: src/one-line.hs
-- one-line (lance putnam)

import Sound.SC3

one_line :: UGen
one_line =
  let lfs = lfSaw AR (mce2 1 0.99) (mce2 0 0.6) * 2000 + 2000
      lfs_t = trunc lfs (mce2 400 600) * mce2 1 (-1)
      f = onePole (mix lfs_t) 0.98
  in pan2 (sinOsc AR f 0) 0 0.1

main :: IO ()
main = audition (out 0 one_line)

File: gr/one-line.fs
\ one-line (ljp)

1 0.99 2 mce 0 0.6 2 mce LFSaw.ar 2000 * 2000 + ( lfs )
400 600 2 mce swap Trunc 1 -1 2 mce * ( lfs_t )
mix 0.98 OnePole ( f )
0 SinOsc.ar 0 0.1 Pan2
play
